R/utils.R

```diff
@@ -25,9 +25,6 @@
 #'  with text placed before the output to put the output into context. For example a title.
 #' @param post_output (`shiny.tag`) optional, text or UI element to be displayed after the module's output,
 #' adding context or further instructions. Elements like `shiny::helpText()` are useful.
-#' @param decorators `r lifecycle::badge("experimental")` (`list` of `teal_transform_module` or `NULL`) optional,
-#' if not `NULL`, decorator for tables or plots included in the module.
```

Comment:
Why this is removed? This was needed so that in each module we added `decorators` parameter automatically to the documentation.

Reply:
It's added per module. [this comment](https://github.com/insightsengineering/teal.modules.general/pull/812#discussion_r1861104970) is a byproduct of this.

Which allows to have the module name specific text. It's a small issue, but it seems odd to have `See tm_<module_name>` in the documentation for the function.

This is a minor thing that we can remove and go back to a shared parameter.

For instance in the `tm_missing_data`.

![image](https://github.com/user-attachments/assets/a23057be-2c6f-4dae-bf02-6e08196d55f1)

```diff
+#' @param decorators `r roxygen_decorators_param("tm_missing_data")`
```